figure out whether akka-parsing needs to be published or removed as dependency

seems like something broke when moving to master

fyi

thats the StackTrace I get if I try to run our example server without parsing https://github.com/MichaelZinsmaier/websocket-bug

Exception in thread "main" java.lang.NoClassDefFoundError: akka/parboiled2/ParserInput
	at akka.http.scaladsl.model.headers.Server$.apply(headers.scala:742)
	at akka.http.ServerSettings$$anonfun$fromSubConfig$1.apply(ServerSettings.scala:55)
	at akka.http.ServerSettings$$anonfun$fromSubConfig$1.apply(ServerSettings.scala:55)
	at scala.Option.map(Option.scala:146)
	at akka.http.ServerSettings$.fromSubConfig(ServerSettings.scala:55)
	at akka.http.ServerSettings$.fromSubConfig(ServerSettings.scala:47)
	at akka.http.impl.util.SettingsCompanion.apply(SettingsCompanion.scala:44)
	at akka.http.impl.util.SettingsCompanion$$anonfun$apply$1.apply(SettingsCompanion.scala:30)
	at scala.collection.MapLike$class.getOrElse(MapLike.scala:128)
	at scala.collection.AbstractMap.getOrElse(Map.scala:59)
	at akka.http.impl.util.SettingsCompanion.apply(SettingsCompanion.scala:29)
	at akka.http.scaladsl.HttpExt.bindAndHandleAsync$default$4(Http.scala:159)
	at server.TestEchoServerApp$.delayedEndpoint$server$TestEchoServerApp$1(TestEchoServerApp.scala:39)
	at server.TestEchoServerApp$delayedInit$body.apply(TestEchoServerApp.scala:17)
	at scala.Function0$class.apply$mcV$sp(Function0.scala:34)
	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:12)
	at scala.App$$anonfun$main$1.apply(App.scala:76)
	at scala.App$$anonfun$main$1.apply(App.scala:76)
	at scala.collection.immutable.List.foreach(List.scala:381)
	at scala.collection.generic.TraversableForwarder$class.foreach(TraversableForwarder.scala:35)
	at scala.App$class.main(App.scala:76)
	at server.TestEchoServerApp$.main(TestEchoServerApp.scala:17)
	at server.TestEchoServerApp.main(TestEchoServerApp.scala)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:144)
Caused by: java.lang.ClassNotFoundException: akka.parboiled2.ParserInput
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 28 more

--- a/akka-parsing/build.sbt
+++ b/akka-parsing/build.sbt
@@ -2,7 +2,6 @@ import akka._
 import com.typesafe.tools.mima.plugin.MimaKeys
 
 AkkaBuild.defaultSettings
-AkkaBuild.dontPublishSettings
 AkkaBuild.experimentalSettings
 Formatting.docFormatSettings
 site.settings

